pkg/kv/kvclient/kvcoord/dist_sender_rangefeed.go

```diff
@@ -280,6 +282,14 @@ func (a *activeRangeFeed) onRangeEvent(
 	a.RangeID = rangeID
 }
 
+func (a *activeRangeFeed) setError(err error) {
```

Comment:
`setLastError`? When I see `setError` it feels like this is suggesting to me that this rangefeed is dead in the water.

Reply:
> `setLastError`? When I see `setError` it feels like this is suggesting to me that this rangefeed is dead in the water.

Done.